packages/web/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "target": "es6",
+      "lib": ["es6", "dom"],
+      "sourceMap": true,
+      "jsx": "react",
+      "moduleResolution": "node",
+    //   "noImplicitAny": true,
```

Comment:
If these are not required, we can remove them

Reply:
This is a good config to follow but we might not comply all rules so better to clean it up. :+1: